UIKitTemplate/UIKitTemplate/Controllers/WebViewController.swift

```diff
@@ -0,0 +1,41 @@
+// WebViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+import WebKit
+
+/// Экран сайта
+class WebViewController: UIViewController, WKNavigationDelegate {
+    // MARK: - Public Properties
+
+    var url = ""
```

Comment:
```suggestion
    private var url = ""
```
заполнять лучше методом, это ведь не делегат и не кложур

Reply:
здесь не заполняю а передаю ссылку